web/management/commands/create_test_data.py

```
@@ -344,7 +371,16 @@ def handle(self, *args, **kwargs):
         now = timezone.now()
         for course in courses:
             for i in range(5):
-                start_time = now + timedelta(days=i * 7)
+                # Randomly decide if the session should be in the past or future
+                in_future = random.choice([True, False])
```

Ruff (0.8.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)